From: Enrico Tassi <gareuselesinge@debian.org>
Date: Wed, 19 Aug 2015 19:44:38 +0200
Subject: Use lua-any to choose the right interpreter version

---
 ldoc.lua | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/ldoc.lua b/ldoc.lua
index e32472c..ccdb65e 100644
--- a/ldoc.lua
+++ b/ldoc.lua
@@ -1,4 +1,5 @@
-#!/usr/bin/env lua
+#!/usr/bin/env lua-any
+-- Lua-Versions: 5.2 5.1
 ---------------
 -- ## ldoc, a Lua documentation generator.
 --
